README.md

```diff
@@ -1,15 +1,31 @@
-# OpenSearch
+<img src="https://github.com/opensearch-project/documentation-website/blob/main/assets/images/logo.svg" height="64px"/>
```

Comment:
Commit into this repo.

Reply:
I'm not sure I understand. Could you please clarify?

Reply:
I think the comment is suggesting the svg file be committed directly to this repo rather than referencing another one to retrieve it.

Reply:
Yup.

Reply:
That would A) duplicate the image; and B) make it impossible to know the source of truth. Would you please state what's the problem with a reference, and if that could be resolved by a less volatile location e.g., `https://www.opensearch.org/logo.svg` ?

Reply:
That external reference to docs has no idea it's being used and is not a stable URL, it will break. IMO anything external will break, but it's not important enough for me, so if you want to use a less volatile location, e.g. https://opensearch.org/brand.html, that's A-OK.